BasicParam:
  framerate: 30
  res_width: 480
  res_height: 480
  vertical_flip: True
  horizontal_flip: False


CalibrationParam:



